modules/installation-initializing-manual.adoc

```diff
@@ -134,12 +134,21 @@ it in the `<installation_directory>`.
 You must name this configuration file `install-config.yaml`.
 ====
 ifdef::restricted[]
+
 ** Unless you use a registry that {op-system} trusts by default, such as
 `docker.io`, you must provide the contents of the certificate for your mirror
 repository in the `additionalTrustBundle` section. In most cases, you must
 provide the certificate for your mirror.
 ** You must include the `imageContentSources` section from the output of the command to
 mirror the repository.
+** The 'ImageContentSourcePolicy' is generated as an output of `oc-mirror` flow (after the mirroring process is finished).
+** `oc-mirror` generates the file called 'ImageContentSourcePolicy' which contains yaml needed to define 'ImageContentSourcePolicy'. 
+Copy the text from this file and paste in your install-config.yaml. 
+** You must run 'oc-mirror' twice. The first time you run `oc-mirror` you will get a full ImageContentSourcePolicy
```

Comment:
I rewrote this section. My main suggestion is here is not to use language like "Please". Do not use please in any instructions. Terms of politeness are superfluous, convey the wrong tone for technical material, and are not regarded the same way in all cultures.

```suggestion
** You must run the 'oc mirror' command twice. The first time you run the`oc mirror` command, you get a full `ImageContentSourcePolicy` file. The second time you run the `oc mirror command`, you only get the difference between the first and second run. Because of this behavior, you must always keep a backup of these files in case you need to merge them into one, complete `ImageContentSourcePolicy` file. Keeping a backup of these two files ensures that you have everything you need for a complete `ImageContentSourcePolicy` file.
```

Reply:
I didn't write that section but I MUST agree with you. Glad to make the changes!